book/macros.txt

```diff
@@ -0,0 +1,37 @@
+\field:{\mathbb{F}}
+\group:{\mathbb{G}}
+\setup:{\textnormal{Setup}}
+\prover:{\mathcal{P}}
+\verifier:{\mathcal{V}}
+\sec:{\lambda}
+\negl:{\textnormal{negl}(\lambda)}
+\pp:{\mathsf{pp}}
+\ip:{\textnormal{IP}}
+\relation:{\mathcal{R}}
+\a:{\mathcal{A}}
+\sim:{\mathcal{S}}
+\tr:{\textnormal{tr}}
+\srs:{\textnormal{SRS}}
+\srwee:{\textnormal{sr-wee}}
+\real:{\textnormal{real}}
+\ideal:{\textnormal{ideal}}
+\weereal:{\textnormal{WEE-real}}
+\weeideal:{\textnormal{WEE-ideal}}
+\oracle:{\mathcal{O}}
+\ch:{\mathsf{Ch}}
+\badch:{\mathsf{BadCh}}
+\adv:{\mathsf{Adv}}
+\bottom:{\perp}
+\alg:{#1_\textnormal{alg}}
+\zero:{\mathcal{O}}
+\dlrel:{\mathsf{dl-rel}}
+\game:{\mathsf{G}}
```

Comment:
```suggestion
\game:{\textnormal{⅁}}
```
(This makes it more distinct from the notation for group elements. Not sure if it'll render correctly but worth a try.)